notebooks/Data_Attribute_Recommendation_Regression_Model_Template.ipynb

```diff
@@ -0,0 +1,1933 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Attribute Recommendation - TechED 2020 INT260\n",
+    "\n",
+    "Getting started with the Python SDK for the Data Attribute Recommendation service."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Business Scenario\n",
+    "\n",
+    "We will consider a business scenario involving product master data. The creation and maintenance of this product master data requires the careful manual selection of the correct categories for a given product from a pre-defined hierarchy of product categories.\n",
+    "\n",
+    "In this workshop, we will explore how to automate this tedious manual task with the Data Attribute Recommendation service."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<video controls src=\"videos/dar_prediction_material_table.mp4\"/>\n",
```

Comment:
I think the video is missing. I am not sure if we should bring it over: at least for the regression template, it might be confusing.

Reply:
Yes, I've included the video only for the hierarchical template now.